components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -9434,7 +9434,8 @@ public void undeployAPIRevisionDeployment(String apiId, String apiRevisionId,
      * @throws APIManagementException if failed to restore APIRevision
      */
     @Override
-    public void restoreAPIRevision(String apiId, String apiRevisionId, String tenantDomain) throws APIManagementException {
+    public void restoreAPIRevision(String apiId, String apiRevisionId, String organizationId)
```

Comment:
shall we use organization as the param

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10508/commits/767280a00ab0096fa01478c333ea3ac8a4984082